src/mapper/pkg/kubefinder/kubefinder.go

```diff
@@ -442,3 +463,53 @@ func (k *KubeFinder) ResolveOtterizeIdentityForService(ctx context.Context, svc
 	dstSvcIdentity.KubernetesService = lo.ToPtr(svc.Name)
 	return dstSvcIdentity, true, nil
 }
+
+func (k *KubeFinder) IsSrcIpLocal(ctx context.Context, ip string) (bool, error) {
```

Comment:
the phrase "local" here is confusing to me. You mean it in the sense of "coming from the local cluster"? So maybe call it "IsSrcIPClusterInternal"?